### PR TITLE
[IE][NGRAPH][BUILD] Enable UNITY build for more targets

### PR DESCRIPTION
**Pull request by vinograd47**
* Avoid symbols clash due to source merging.
* Tested on Ubuntu 20.04 with GCC 9.3.0.

### CODE REVIEW

File: ngraph/core/src/op/add.cpp [hunk 1]
```diff
@@ -53,7 +53,7 @@ shared_ptr<Node> ngraph::operator+(const Output<Node>& arg0, const Output<Node>&
     return make_shared<op::Add>(arg0, arg1);
 }
 
-namespace
+namespace add
```
Comment (ilya-lavrenov) [inline ngraph/core/src/op/add.cpp:56]: why unnamed namespaces don't work?

Comment (vinograd47) [reply ngraph/core/src/op/add.cpp:56]: Unity build merges several source files into single one. So you would get one single source file with several unnamed namespaces with the same symbols. It fails to compile. Thus I have to use different symbol names even for internal stuff.

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by vinograd47** [comment]
@ilyachur Is there any way to get actual "code style" issue for nGraph? I have Ubuntu 20.04 and `clang-format-3.9` is not available there.

**Comment by ilyachur** [comment]
> @ilyachur Is there any way to get actual "code style" issue for nGraph? I have Ubuntu 20.04 and `clang-format-3.9` is not available there.

Sure 😄 I also don't have a `clang-3.9` on masOS. Please click on Details for code style check and click on artefacts. Artefacts contains an [archive](https://github.com/openvinotoolkit/openvino/suites/1313681508/artifacts/20785201) with a patch.

**Review by ilyachur — APPROVED** [review]

**Pull request merged**